components/tooltip/script.js

```diff
@@ -132,6 +141,11 @@ class Tooltip extends HTMLElement {
             .catch(err => console.error(err));
     }
 
+    onWindowResize() {
+        this.setPosition(window.scrollX, window.scrollY);
+        this.uncheckedOrientations = TOOLTIP_POSITIONS;

```

Comment:
Maybe assign it as a copy like `this.uncheckedOrientations = [...TOOLTIP_POSITIONS]` to prevent changing the `TOOLTIP_POSITIONS` reference with a mutable operation? I know you make a new array about the `this.uncheckedOrientations` in `getVisiblePosition` but someone may hit this problem.

Reply:
Changed.